AutonomousDemo/Velocity.cs

```diff
@@ -9,10 +9,10 @@ namespace AutonomousDemo
     public class Velocity
     {
         public double Speed { get; set; } = 0;
-        public double Direction { get; set; }  //radians
+        public double Direction { get; set; }  
 
 
-        public Velocity(double speed, double direction = 1.5708)
+        public Velocity(double speed, double direction = Math.PI/2.0)
```

Comment:
Just to note, presumably ANY direction that doesn't change represents driving in a straight line...